tests/bin/draft/run.t

```diff
@@ -0,0 +1,128 @@
+We need a basic opam project skeleton
+
+    $ cat > CHANGES.md << EOF \
+    > ## 0.1.0\
+    > \
+    > - Some other feature\
+    > \
+    > ## 0.0.0\
+    > \
+    > - Some feature\
+    > EOF
+    $ cat > whatever.opam << EOF \
+    > opam-version: "2.0"\
+    > homepage: "https://github.com/foo/whatever"\
+    > dev-repo: "git+https://github.com/foo/whatever.git"\
+    > description: "whatever"\
+    > EOF
+    $ touch README
+    $ touch LICENSE
+    $ cat > dune-project << EOF \
+    > (lang dune 2.4)\
+    > (name whatever)\
+    > EOF
+
+We need to set up a git project for dune-release to work properly
+
+    $ cat > .gitignore << EOF \
+    > _build\
+    > .formatted\
+    > .mdx\
+    > /dune\
+    > run.t\
+    > EOF
+    $ git init 2> /dev/null > /dev/null
+    $ git config user.name "dune-release-test"
+    $ git config user.email "pseudo@pseudo.invalid"
+    $ git add CHANGES.md whatever.opam dune-project README LICENSE .gitignore
+    $ git commit -m "Initial commit" > /dev/null
+    $ dune-release tag -y
+    [-] Extracting tag from first entry in CHANGES.md
+    [-] Using tag "0.1.0"
+    [+] Tagged HEAD with version 0.1.0
+
+We do the whole dune-release process
+
+(1) distrib
+
+    $ dune-release distrib --dry-run
+    [-] Building source archive
```

Comment:
The output of this command can be ignored, we just want to create the tarball for the next command right?

Reply:
Indeed, done!